### PR TITLE
Handle no schema cases.

### LINKED ISSUE #83: Gracefully handle when no schema is provided
At the moment this causes a crash:

```
$ graphql-schema-linter
It looks like you may have hit a bug in graphql-schema-linter.

It would be super helpful if you could report this here: https://github.com/cjoudrey/graphql-schema-linter/issues/new

TypeError: Must provide Source. Received: null
    at parse (/Users/cjoudrey/.config/yarn/global/node_modules/graphql/language/parser.js:43:11)
    at validateSchemaDefinition (/Users/cjoudrey/.config/yarn/global/node_modules/graphql-schema-linter/lib/validator.js:21:30)
    at run (/Users/cjoudrey/.config/yarn/global/node_modules/graphql-schema-linter/lib/runner.js:46:56)
    at Object.<anonymous> (/Users/cjoudrey/.config/yarn/global/node_modules/graphql-schema-linter/lib/cli.js:15:32)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Function.Module.runMain (module.js:676:10)
```

### LINKED ISSUE #84: Gracefully handle when an non-existing schema file is provided
At the moment this causes a crash:

```
$ graphql-schema-linter invalid.graphql
It looks like you may have hit a bug in graphql-schema-linter.

It would be super helpful if you could report this here: https://github.com/cjoudrey/graphql-schema-linter/issues/new

Error: ENOENT: no such file or directory, open 'invalid.graphql'
    at Object.fs.openSync (fs.js:646:18)
    at fs.readFileSync (fs.js:551:33)
    at getSchemaFromFile (/Users/cjoudrey/.config/yarn/global/node_modules/graphql-schema-linter/lib/configuration.js:191:31)
    at /Users/cjoudrey/.config/yarn/global/node_modules/graphql-schema-linter/lib/configuration.js:196:22
    at Array.reduce (<anonymous>)
    at getSchemaSegmentsFromFiles (/Users/cjoudrey/.config/yarn/global/node_modules/graphql-schema-linter/lib/configuration.js:195:16)
    at Configuration.getSchema (/Users/cjoudrey/.config/yarn/global/node_modules/graphql-schema-linter/lib/configuration.js:90:24)
    at run (/Users/cjoudrey/.config/yarn/global/node_modules/graphql-schema-linter/lib/runner.js:41:30)
    at Object.<anonymous> (/Users/cjoudrey/.config/yarn/global/node_modules/graphql-schema-linter/lib/cli.js:15:32)
    at Module._compile (module.js:635:30)
```

### PR DESCRIPTION
**Pull request by ruiaraujo**
Closes #83 
Closes #84

### CODE REVIEW

File: src/configuration.js [hunk 1]
```diff
@@ -39,15 +38,15 @@ export class Configuration {
...
       this.schema = getSchemaFromFileDescriptor(this.stdinFd);
       this.sourceMap = new SourceMap({ stdin: this.schema });
     } else if (this.options.schemaPaths) {
-      var expandedPaths = expandPaths(this.options.schemaPaths);
-      var segments = getSchemaSegmentsFromFiles(expandedPaths);
-
+      const expandedPaths = expandPaths(this.options.schemaPaths);
+      const segments = getSchemaSegmentsFromFiles(expandedPaths);
+      if (Object.keys(segments).length === 0) {
+        return this.schema;
```
Comment (cjoudrey) [inline src/configuration.js]: Should this be `return null;`? I think that's essentially what will happen, but I thought returning `this.schema` was a bit confusing.

**Comment by cjoudrey** [comment]
Thanks for the contribution and for the code cleanup @ruiaraujo! 🎉 

I'm a bit puzzled by the test failure. Can you think of why this might be happening?

**Comment by ruiaraujo** [comment]
It was nothing, the empty file is getting skipped.

**Comment by cjoudrey** [comment]
Thanks for the contribution, I'll 🚢 this in the next release.

**Comment by cjoudrey** [comment]
This has shipped in v0.0.29. Thanks again! 😄

**Pull request merged**